var/spack/repos/builtin/packages/python/package.py

```diff
@@ -58,73 +58,182 @@ class Python(Package):
     license("0BSD")
 
     version("3.13.2", sha256="b8d79530e3b7c96a5cb2d40d431ddb512af4a563e863728d8713039aa50203f9")
-    version("3.13.1", sha256="1513925a9f255ef0793dbf2f78bb4533c9f184bdd0ad19763fd7f47a400a7c55")
-    version("3.13.0", sha256="12445c7b3db3126c41190bfdc1c8239c39c719404e844babbd015a1bc3fafcd4")
```

Comment:
I'm personally fine with this. Wondering if other users might need older patch versions of the latests stable release, and we should use the same strategy as https://github.com/spack/spack/pull/49113